webui/react/src/components/Navigation.module.scss

```diff
@@ -38,3 +38,18 @@
     }
   }
 }
+
+// If clusterMessage is present on page, override some heights to make room
+
+.clusterMessage {
+  --cluster-message-height: 40px;
+  height: calc(100vh - var(--cluster-message-height));
+
+  @media only screen and (max-width: $breakpoint-mobile) {
+    & > main {
+      height: calc(
+        var(--vh, 1vh) * 100 - var(--nav-bottom-bar-height) - var(--cluster-message-height)
```

Comment:
why not just `100vh` instead of `var(--vh, 1vh) * 100`?

Reply:
I'm not sure what the var `--vh` is or how to figure that out. I believe I copied this from line 37 above. I'm happy to change this, but don't feel like I know enough what I'm doing to confidently do it and know what to test to make sure I'm not breaking anything. Any advice? Or is it fine to leave it like line 37?

Reply:
`--vh` is defined here 

https://github.com/determined-ai/determined/blob/aa6521bbc54dcd2205bdef0921b11adac3b3ea27/webui/react/src/utils/browser.ts#L8-L15

looks like its introduced to fix the mobile view height, but i don't see the issue anymore (`--vh` is tech debt now?), so i think `100vh` should be fine, but up to you

Reply:
Thank you! Good to know that ts consts can be CSS vars! I had no idea.